motr/motr-pub.api

```diff
@@ -469,3 +469,4 @@ m0_error_printf
 *__gcov_flush
 *__gcov_reset
 *__gcov_sort_n_vals
+print_pi
```

Comment:
That's a wrong name for a public api.

Reply:
We have removed this part of code long back, looks like github not rendered latest code.